site/app/config/services.yml

```diff
@@ -3,9 +3,18 @@
 parameters:
 #    parameter_name: value
 
+imports:
+    - { resource: legacy_aliases.yml }
+
 services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
     # Twig date extension - http://twig-extensions.readthedocs.io/en/latest/date.html
-    twig.extension.date:
-       class: Twig_Extensions_Extension_Date
-       tags:
-            - { name: twig.extension }
+    #twig.extension.date: # html template
+    #    class: Twig_Extensions_Extension_Date # getagged class
+    Twig\Extensions\DateExtension:
+        autowire: false
```

Comment:
... and add back the comment I proposed (or a similar one) explaining why we have to set autowire to false in this case.